cmake/qemu-command.cmake

```diff
@@ -12,7 +12,7 @@ else()
   endif()
   set(QEMU_COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR}
     "-one-insn-per-tb"
-    "-L" "${CMAKE_BINARY_DIR}/external/glibc/sysroot/usr/"
```

Comment:
Does the sysroot not include the headers anymore? Why are we removing the usr suffix?

Reply:
That seems fine then